src/nanoarrow/schema_test.cc

```diff
@@ -1192,9 +1197,7 @@ TEST(SchemaViewTest, SchemaViewInitNestedUnion) {
   EXPECT_EQ(schema_view.layout.element_size_bits[0], 8);
   EXPECT_EQ(schema_view.layout.element_size_bits[1], 32);
   EXPECT_EQ(schema_view.layout.element_size_bits[2], 0);
-  EXPECT_EQ(
-      std::string(schema_view.union_type_ids.data, schema_view.union_type_ids.n_bytes),
-      std::string("0"));
+  EXPECT_EQ(std::string(schema_view.union_type_ids), std::string("0"));
```

Comment:
nit: if they're both null-terminated strings, you can `EXPECT_STREQ` (not a big deal here though)